org.kde.kweather.json

```diff
@@ -0,0 +1,56 @@
+{
+    "id": "org.kde.kweather",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kweather",
+    "rename-icon": "kweather",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--system-talk-name=org.freedesktop.GeoClue2"
+    ],
+
+    "modules": [
+        {
+            "name": "kweathercore",
+            "config-opts": [ "-DBUILD_TESTING=OFF", "-DCMAKE_BUILD_TYPE=Release" ],
```

Comment:
"-DCMAKE_BUILD_TYPE=Release" > I don't think those are needed

Reply:
Is it added by default?

Reply:
`-DCMAKE_BUILD_TYPE=RelWithDebInfo` is prefered. The debug info are extracted into their own package.

Reply:
Oh, alright!
I can add it to the other packages I'm working on then too.